flink-ml-python/pyflink/ml/core/param.py

```diff
@@ -333,6 +334,17 @@ def __init__(self, name: str, description: str, default_value: Optional[str],
         super(StringParam, self).__init__(name, str, "str", description, default_value, validator)
 
 
+class VectorParam(Param[Vector]):
```

Comment:
Let's add test cases for this parameter in python as well. Relevant test cases seem to be in `test_stage.py`.